service/history/interface.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017-2020 Uber Technologies Inc.
```

Comment:
to be consistent you can move out the interface in matching/handler.go to separate file